enterprise/api/license.go

```diff
@@ -1,17 +1,62 @@
 package api
 
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/bytebase/bytebase/api"
+)
+
+// validPlans is a string array of valid plan types.
+var validPlans = []string{
+	api.TEAM.String(),
+	api.ENTERPRISE.String(),
+}
+
 // License is the API message for enterprise license.
 type License struct {
-	Audience      string `jsonapi:"attr,audience"`
-	InstanceCount int    `jsonapi:"attr,instanceCount"`
-	ExpiresTs     int64  `jsonapi:"attr,expiresTs"`
-	Plan          string `jsonapi:"attr,plan"`
+	Audience      string       `jsonapi:"attr,audience"`
+	InstanceCount int          `jsonapi:"attr,instanceCount"`
+	ExpiresTs     int64        `jsonapi:"attr,expiresTs"`
+	Plan          api.PlanType `jsonapi:"attr,plan"`
+}
+
+// Valid will check if license expired or has correct plan type.
+func (l *License) Valid() error {
+	if l.ExpiresTs <= time.Now().Unix() {
+		return fmt.Errorf("license has expired at %v", time.Unix(l.ExpiresTs, 0))
```

Comment:
Since we already use time.Unix(l.ExpiresTs, 0), we can do the comparison in the other way.

expireTime := time.Unix(l.ExpiresTs, 0)
if expireTime.Before(time.Now()) {
  return fmt.Errorf("license has expired at %v", expireTime)
}

Reply:
Sure, updated. But I'm wondering what's the difference between seconds compare VS time compare?

Reply:
two benefits
 - Before/After() are more human readable as time to time comparison.
 - we can save/centralize a variable for expireTime.